libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -650,7 +1262,71 @@ async def wait(
         webhook: Optional[str] = None,
         multitask_strategy: Optional[MultitaskStrategy] = None,
     ) -> Union[list[dict], dict[str, Any]]:
-        """Create a run, wait for and return the final state."""
+        """Create a run, wait for and return the final state.
+
+        Args:
+            thread_id (optional,str): the thread ID to create the run on. Defaults to None.
+                If None a random UUID will be generated.
```

Comment:
this is actually not the point in this case -- this is for creating stateless runs. e.g. if you pass thread_id = None, we won't store the thread state but will still return the output